dataland-frontend/src/components/forms/CreateCompany.vue

```diff
@@ -37,40 +32,42 @@ import {CompanyDataControllerApi} from "@/../build/clients/backend";
 import SuccessUpload from "@/components/ui/SuccessUpload";
 import {DataStore} from "@/services/DataStore";
 import backend from "@/../build/clients/backend/backendOpenApi.json";
-import CardWrapper from "@/components/wrapper/CardWrapper";
 import FailedUpload from "@/components/ui/FailedUpload";
-
+import Card from 'primevue/card';
+import Message from 'primevue/message';
 const api = new CompanyDataControllerApi()
 const contactSchema = backend.components.schemas.CompanyInformation
 const dataStore = new DataStore(api.postCompany, contactSchema)
 
 const createCompany = {
   name: "CreateCompany",
-  components: {FailedUpload, CardWrapper, FormKit, FormKitSchema, SuccessUpload},
+  components: {FailedUpload, Card, Message, FormKit, FormKitSchema, SuccessUpload},
 
   data: () => ({
-    enableClose: false,
+    action: false,
     data: {},
     schema: dataStore.getSchema(),
     model: {},
     loading: false,
     response: null,
-    errorOccurence: false
+    count: 0
```

Comment:
Ist es Company Count? das könnte umbennant werden denke ich

Reply:
nein das wieder nur der count für die unique keys in message, siehe anderer comment

Reply:
dann könnte er eventuel messageCount oder so heißen, oder?

Reply:
geändert